bitcoin.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var request = require('request');
+
+var bitcoin = module.exports = {
+
+	createAddress : function (add1, add2, add3) {
```

Comment:
This function is not really necessary. Can you merge it into the other function (`getFunds`)?